model/src/main/scala/hmda/model/institution/InstitutionStatus.scala

```diff
@@ -0,0 +1,17 @@
+package hmda.model.institution
+
+import enumeratum.{ Enum, EnumEntry }
+
+/**
+ * The status of a financial institution
+ */
+sealed abstract class InstitutionStatus(override val entryName: String) extends EnumEntry
+
+object InstitutionStatus extends Enum[InstitutionStatus] {
+
+  val values = findValues
+
+  case object Active extends InstitutionStatus("active")
+  case object Inactive extends InstitutionStatus("inactive")
```

Comment:
Similar to comment above, there was an initial attempt at this [here](https://github.com/cfpb/hmda-platform/blob/master/model/src/main/scala/hmda/model/fi/Institution.scala#L3), which results in duplication of code